website/package.json

```diff
@@ -88,22 +87,21 @@
     "@chakra-ui/storybook-addon": "^4.0.16",
     "@faker-js/faker": "^7.6.0",
     "@stardazed/streams-text-encoding": "^1.0.2",
-    "@storybook/addon-actions": "^6.5.15",
-    "@storybook/addon-essentials": "^6.5.16",
-    "@storybook/addon-interactions": "^6.5.16",
-    "@storybook/addon-links": "^6.5.16",
-    "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/builder-webpack5": "^6.5.16",
-    "@storybook/manager-webpack5": "^6.5.16",
-    "@storybook/react": "^6.5.16",
-    "@storybook/testing-library": "^0.0.13",
+    "@storybook/addon-actions": "^7.0.5",
+    "@storybook/addon-essentials": "^7.0.5",
+    "@storybook/addon-interactions": "^7.0.5",
+    "@storybook/addon-links": "^7.0.5",
+    "@storybook/nextjs": "^7.0.5",
+    "@storybook/react": "^7.0.5",
+    "@storybook/testing-library": "^0.1.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@types/accept-language-parser": "^1.5.3",
     "@types/node": "^18.15.10",
     "@types/react": "18.0.26",
     "@types/react-syntax-highlighter": "^15.5.6",
     "@typescript-eslint/eslint-plugin": "^5.56.0",
+    "assert": "^2.0.0",
```

Comment:
This is required since webpack 5 (the default in storybook v7) doesn't include a polyfill for assert by default anymore.